scripts/main.js

```diff
@@ -285,8 +284,6 @@ function submitImgFormHandler(elem, event) {
       $("span[id='tabLabel_"+uid+"']").html(
           makeTabLabelContent(uid)
       );
-      $("#deleteBtn_"+uid).click( function(event){deleteImgForm(this);} );
```

Comment:
This was a duplicate binding (other one [here](https://github.com/gd-codes/mc-pixelart-maker/blob/main/scripts/dynamicHtml.js#L155)), causing the delete function and alert() to execute twice.
It seems to be a pre-existing bug, but somehow only started appearing from now?
Deletion still works after removing this line.